lib/widgets/components/product_card_special.dart

```diff
@@ -0,0 +1,96 @@
+import 'package:flutter/material.dart';
+
+class ProductCardSpecial extends StatelessWidget {
+  final String image;
+  final String title;
+  final String description;
+  final String price;
+
+  const ProductCardSpecial({
+    super.key,
+    required this.image,
+    required this.title,
+    required this.description,
+    required this.price,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    return Card(
+      child: Container(
+        padding: const EdgeInsets.fromLTRB(16, 16, 0, 16),
+        decoration: BoxDecoration(
+          borderRadius: BorderRadius.circular(16),
+          gradient: const LinearGradient(
+            colors: [Color(0xFFFB565D), Color(0xFFAB1A20)],
+          ),
+          boxShadow: [
+            BoxShadow(
+              color: Colors.black.withOpacity(0.04),
+              blurRadius: 16,
+              spreadRadius: 0,
+            ),
+          ],
+        ),
+        child: Row(
+          mainAxisAlignment: MainAxisAlignment.spaceBetween,
+          crossAxisAlignment: CrossAxisAlignment.start,
+          children: [
+            Column(
+              crossAxisAlignment: CrossAxisAlignment.start,
+              children: [
+                Text(
+                  title,
+                  overflow: TextOverflow.ellipsis,
+                  style: const TextStyle(
+                    color: Color(0xFFFFFFFF),
+                    fontSize: 16,
+                    fontWeight: FontWeight.w600,
+                    fontFamily: 'Inter',
+                    fontStyle: FontStyle.normal,
+                  ),
+                ),
+                const SizedBox(height: 8),
+                Text(
+                  description,
+                  overflow: TextOverflow.ellipsis,
+                  style: const TextStyle(
+                    color: Color(0xFFFFFFFF),
+                    fontSize: 14,
+                    fontWeight: FontWeight.w400,
+                    fontFamily: 'Inter',
+                    fontStyle: FontStyle.normal,
+                  ),
+                ),
+                const SizedBox(height: 16),
+                Text(
+                  price,
+                  overflow: TextOverflow.ellipsis,
+                  style: const TextStyle(
+                    color: Color(0xFFFFFFFF),
+                    fontSize: 20,
+                    fontWeight: FontWeight.w700,
+                    fontFamily: 'Inter',
+                    fontStyle: FontStyle.normal,
+                  ),
+                ),
+              ],
+            ),
+            const SizedBox(width: 32),
+            Container(
```

Comment:
Is container needed for height and width constraints on image? Use whole numbers.